ft/txn/rollback.cc

```diff
@@ -92,7 +92,7 @@ static inline PAIR_ATTR make_rollback_pair_attr(long size) {
 PAIR_ATTR
 rollback_memory_size(ROLLBACK_LOG_NODE log) {
     size_t size = sizeof(*log);
-    if (&log->rollentry_arena) {
+    if (1) { // if (&log->rollentry_arena) {
```

Comment:
Looks like it was introduced here https://github.com/percona/PerconaFT/pull/307, so both need fixing.